general_itests/steps/deployments_json_steps.py

```diff
@@ -57,6 +60,30 @@ def step_impl_given(context):
     context.expected_commit = commit.id
 
 
+@when(u'paasta stop is run against the repo')
```

Comment:
It might be nice to parameterize this step so that you can run any paasta command against it.